build.gradle

```diff
@@ -232,6 +241,16 @@ task generateApi {
     }
 }
 
+task generateVersionFile {
```

Comment:
What will actually call this task?

Reply:
The task is called automatically during `assemble`.